events/status.go

```diff
@@ -90,9 +90,9 @@ func (w *Watcher) Cleanup(err error) error {
 	select {
 	case werr := <-w.errCh:
 		if err != nil {
-			w.warningf("Deploy() failed: %v", err)
+			w.warningf("Create() failed: %v", err)
```

Comment:
this is a generic watcher (as is the pods watcher), so this error message should be different

Reply:
In particular, something other than Create() might call it.

Reply:
Fixed to have a generic error message instead, thanks